config/base.yml

```diff
@@ -621,12 +621,18 @@ Naming/VariableName:
   Enabled: true
   EnforcedStyle: snake_case
 
+Performance/ArraySemiInfiniteRangeSlice:
+  Enabled: true
```

Comment:
FYI, `Performance/ArraySemiInfiniteRangeSlice` cop will be removed in the future, so it is recommended to disable.
https://github.com/rubocop-hq/rubocop-performance/issues/198

This cop is disabled by default and it's best not to list it in config/base.yml in case it's removed.

Reply:
Thanks for the info!